packages/main/cypress/specs/Button.cy.tsx

```diff
@@ -303,12 +302,19 @@ describe("Accessibility", () => {
 		cy.get("@button")
 			.shadow()
 			.find("button")
-			.should("have.attr", "aria-describedby", hiddenTextTypeId);
+			.should("have.attr", "aria-description", "Warning");
+	});
+
+	it("accessibleDescription in combination with design property applied on the button tag", () => {
+		cy.mount(<Button design="Negative" accessibleDescription="Decline">Content</Button>);
+
+		cy.get("[ui5-button]")
+			.as("button");
 
 		cy.get("@button")
 			.shadow()
-			.find(`span[id="${hiddenTextTypeId}"]`)
-			.should("exist");
+			.find("button")
+			.should("have.attr", "aria-description", "Negative Action Decline");
```

Comment:
Add following import
```
import {
	BUTTON_ARIA_TYPE_REJECT,
} from "../../src/generated/i18n/i18n-defaults.js";
```

and text with:
```
.should("have.attr", "aria-description", `${BUTTON_ARIA_TYPE_REJECT.defaultText} Decline`);
```

This will prevent test from failing if the texts are changed due translations